### DISCUSSION TITLE
Add websocket benchmark + memory used + cpu % on number of connections per second etc

### DISCUSSION THREAD

**Discussion by ouvaa** [TechEmpower/FrameworkBenchmarks#8780] [Ideas]
1. Add websocket benchmark on number of connections per second etc.
2. add memory used
3. add cpu % used 
for all benchmarks

**Comment by joanhey** [comment]
For now I think that websocket is not the direction of this benchmark.

And for the memory and cpu used, you can check this url:
https://ajdust.github.io/tfbvis/

Thank you.
